airbyte-integrations/connectors/source-oracle/src/main/resources/spec.json

```diff
@@ -0,0 +1,41 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/oracle",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Oracle Source Spec",
+    "type": "object",
+    "required": ["host", "port", "database", "username"],
+    "additionalProperties": false,
+    "properties": {
+      "host": {
+        "title": "Host",
+        "description": "Hostname of the database.",
+        "type": "string"
+      },
+      "port": {
+        "title": "Port",
+        "description": "Port of the database.",
+        "type": "integer",
+        "minimum": 0,
+        "maximum": 65536,
+        "default": 1521,
+        "examples": ["1521"]
+      },
+      "sid": {
```

Comment:
i'm not really familiar with what this is. should it be a required field?

Reply:
It's similar to the database name in Postgres.
SID = the unique name of your DB,

Reply:
should it be required? or is it okay if it is left blank?

Reply:
It's required